__tests__/app.test.js

```diff
@@ -167,27 +167,25 @@ describe('GET /api/articles/:article_id/comments', ()=>{
     test('should return a 200 status code with an empty array if the article_id is valid but does not exist',()=>{
```

Comment:
Don't forget to update the test description if testing a different status code

Reply:
Done